### ISSUE TITLE
paper.getContentBBox() not working when cell changes under IE11

### ISSUE THREAD

**Issue by fudanglp** [clientIO/joint#176]
Possible caused by:
IE11 Element.getBoundingClientRect() does not support SVG Element well.

**Comment by kumilingus** [comment]
It seems this happens when an `svgPath` has invalid or no `d` attribute.

IE returns an empty bounding box.
```
<g>
  <path d />
  <rect width="100" height="100" x="10" y="10" />
</g>
```

IE returns { x: 10, y: 10, width: 100, height: 100 }
```
<g>
  <path d="M 10 10 100 100" />
  <rect width="100" height="100" x="10" y="10" />
</g>
```

**Comment by kumilingus** [comment]
Please always make sure there is a valid `d` attribute on all `SVGPath` elements.
Fixed in d45983f65d02cf10a510a281032848ba2fffce75.
